localflavor/ir/forms.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""AR-specific Form helpers."""
+
+from __future__ import unicode_literals
+
+from django.forms import ValidationError
+from django.forms.fields import CharField, RegexField, Select
+from django.utils.translation import ugettext_lazy as _
+
+from localflavor.compat import EmptyValueCompatMixin
+
+from .ir_provinces import PROVINCE_CHOICES
```

Comment:
The tests are failing because you have imports that aren't used. Try replacing the imports with this:

```
from __future__ import unicode_literals

from django.forms.fields import Select

from .ir_provinces import PROVINCE_CHOICES
```